src/matrix/lkj.jl

```diff
@@ -186,14 +186,16 @@ end
 #  for 1 / c₀, even if they say that it is not.
 #  -----------------------------------------------------------------------------
 
-function lkj_onion_loginvconst(d::Integer, η::Real)
+function lkj_onion_loginvconst(d::Integer, η::T) where {T <: Real}
```

Comment:
The functions below `lkj_onion_loginvconst_uniform_even` and `lkj_onion_loginvconst_uniform_odd` have to be fixed as well, otherwise `lkj_logc0` becomes type unstable it seems.